lib/kuiq/model/job_manager.rb

```diff
@@ -0,0 +1,92 @@
+require "kuiq/model/job"
+
+module Model
+  class JobManager
+    REDIS_PROPERTIES = %w[redis_version uptime_in_days connected_clients used_memory_human used_memory_peak_human]
+    
+    attr_accessor :jobs, :polling_interval
+    attr_reader :redis_url, :current_time, :docs_url, :locale, :locale_url, :redis_info
+  
+    def initialize
+      @current_time = Time.now
+      @jobs = []
+      @polling_interval = 5
+      @redis_url = Sidekiq.redis { |c| c.config.server_url }
+      @current_time = Time.now.utc
+      @docs_url = "https://github.com/sidekiq/sidekiq/wiki"
+      @locale_url = "https://github.com/sidekiq/sidekiq/"
+      @locale = "en"
+      @redis_info = Sidekiq.default_configuration.redis_info
+    end
+  
+    def stats
+      # do not cache in a variable to ensure getting the latest values when calling methods
+      # off of the Status object (e.g. when calling stats.processed)
```

Comment:
The issue here is that S::Stats goes to Redis for each metric. We want to cache the Stats so all of the status counts can be updated with just one network round trip. In other words the refresh timer should invalidate the cached S::Stats so `stats` can refresh the cached values from Redis every N seconds.